auth/user_mgt.go

```diff
@@ -728,6 +947,91 @@ func (c *baseClient) DeleteUser(ctx context.Context, uid string) error {
 	return err
 }
 
+// A DeleteUsersResult represents the result of the DeleteUsers() call.
+type DeleteUsersResult struct {
+	// The number of users that were deleted successfully (possibly zero). Users
+	// that did not exist prior to calling DeleteUsers() will be considered to be
+	// successfully deleted.
+	SuccessCount int
+
+	// The number of users that failed to be deleted (possibly zero).
+	FailureCount int
+
+	// A list of describing the errors that were encountered
```

Comment:
Delete "of" if this list actually describes the errors.  Otherwise, just "A list of errors encountered..."

Suggest checking whether this was copied into similar comment in other SDKs, and we just missed it :)

Reply:
Oh; the type somehow got dropped. (Which is mildly redundant info in most ports since you can get it from the type signature; python being the obvious exception.) The other ports seem ok.

I've added the type to match the other ports. Alternatively, we could just remove the type everywhere (except python).